hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1634,11 +1637,13 @@ private void renameBlobDir(final Path source,
     }
     renameBlobExecutorService.shutdown();
 
+    tracingContext.setOperatedBlobCount(renamedBlob.get() + 1);
```

Comment:
Since we are already doing an increment and get, why +1 here ?

Reply:
Increment is being done for blobs in renamed in parallel

This +1 is for blob which would contain the count value in CID.